### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by AutumnFoto**
## Description

updated read me

## Related Issue
#20 

## Motivation and Context

read me needed
## How Can This Be Tested?

git fetch update-readme

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by chris-calhoun — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
-[Chris Calhoun](https://github.com/chris-calhoun)
-[Autumn Fotopoulos](https://github.com/AutumnFoto)
+## Contributors
+* [Jordan Smith](https://github.com/JSmith989)
+* [Bailey Dennie](https://github.com/BaileyEDennis)
+* [Chris Calhoun](https://github.com/chris-calhoun)
+* [Autumn Fotopoulos](https://github.com/AutumnFoto)
 
-#Teach Used
+### Teach Used
```
Comment (chris-calhoun) [inline README.md:22]: *Tech

**Review by chris-calhoun — APPROVED** [review]
Left a comment but other looks good 👍

**Review by JSmith989 — APPROVED** [review]
+1

**Pull request merged**